explainer/Explainer.md

```diff
@@ -187,18 +194,21 @@ use cases like a Smart City where there
 might be thousands of Things to select from.
 However, it can also be used in a Smart Home with only a few dozen Things.
 
-Self-description is another exploration mechanism that
-provides the ability of an HTTP-based Thing to
-return a TD directly.  There is also an Introduction mechanism that allows
+A TD Service is another exploration mechanism that allows a single
```

Comment:
```suggestion
A TD server is another exploration mechanism that allows a single
```
?

Reply:
good catch, fixed.  But I used uppercase to be consistent with the spec.